crates/mock-download-server/src/server.rs

```diff
@@ -51,9 +67,171 @@ impl MockServer {
         self.served_requests.load(Ordering::SeqCst)
     }
 
+    pub fn release_package(&self) -> HashMap<(String, String, String), Vec<u8>> {
```

Comment:
I believe this is dead code? If not, it's misnamed (it returns a hashmap of all release packages, not just one).